### DISCUSSION TITLE
Dịch "autoencoder"

### DISCUSSION THREAD

**Discussion by trgiangdo** [d2l-ai/d2l-vi#2751] [Terminology]
Trong PR #2695 anh @cuongvng có đề xuất dịch `autoencoder` thành `bộ mã hoá tự động`.
Em đề xuất dịch là `bộ tự mã hoá`.
Không biết ý kiến mọi người thế nào.

cc: @duythanhvn @minhduc0711 @cuongvng @lkhphuc @davidnvq @rootonchair

**Comment by nguyenthaibinh** [comment]
Mình thích cách dịch "bộ tự mã hoá" hơn. Từ "auto" ở đây có thể hiểu là "với chính nó"/"tự nó" hơn là "tự động" (automation/automatic). Trong thuật ngữ Toán học có: "autoregression" => tự hồi quy (1 chuỗi tự hồi quy ra chính nó), "autocorrelation" => tự tương quan (tương quan với chính nó).
Mình hiểu "Autoencoder" là: encode bằng cách khôi phục chính nó.
Hơn nữa nó ngắn gọn hơn "bộ mã hoá tự động".

**Reply by thanhcsf** [reply]
Vậy chốt từ này như đề xuất của @trGiang99 nhé anh em.
